github/gh-collaboration.qmd

```diff
@@ -13,6 +13,9 @@
 
 ## Making a draft pull request
 
+After making and committing changes as described above, navigate to the Pull requests tab. Click "New pull request" which will produce a "Comparing changes" page with four drop-down lists. The leftmost two drop-down lists refer to the target repository of the pull request and should be set to NBISweden/Training-Tech-shorts, followed by the target branch. The two rightmost drop-down lists refer to the pull request source and should point to your repository and, importantly, the branch that you are editing (and make sure it matches the target branch!). By default, only the branches of the target repository are shown. To find the updated branch from the forked repository, one has to click on "If you need to, you can also compare across forks.". Once done, click "Create pull request" which will generate a pull request page where your review partner can make comments on your PR. Finally, change the green drop-down button "Create pull request" to "Create draft pull request" and you're done!

```

Comment:
>  Once done, click "Create pull request" which will generate a pull request page where your review partner can make comments on your PR. Finally, change the green drop-down button "Create pull request" to "Create draft pull request" and you're done!

Sorry, my previous comment on this still stands. Perhaps swap these two sentences.